README.md

```diff
@@ -59,36 +59,45 @@ We are breaking down the various parts that we need to migrate to Swift in the s
+|                     | `interceptContext`                                  | ✅     |
+|                     | `logger`                                            | ✅     |
 |                     | `onActivation`                                      |        |
-|                     | `onReady`                                           |        |
-|                     | `onRefresh`                                         |        |
-|                     | `onUpdate`                                          |        |
-|                     | `refreshInternal`                                   |        |
-|                     | `stickyFeatures`                                    |        |
-| Instance methods    | `setDatafile`                                       |        |
-|                     | `setStickyFeatures`                                 |        |
-|                     | `getRevision`                                       |        |
-|                     | `getFeature`                                        |        |
-|                     | `getBucketBy`                                       |        |
-|                     | `getBucketValue`                                    |        |
-|                     | `isReady`                                           |        |
+|                     | `onReady`                                           | ✅     |
+|                     | `onRefresh`                                         | ✅     |
+|                     | `onUpdate`                                          | ✅     |
+|                     | `refreshInternal`                                   | ✅     |
+|                     | `stickyFeatures`                                    | ✅     |
+|                     |                                                     |        |
+| Instance methods    | `constructor` missing fetch datafile content        | ⚠️      |
+|                     | `setDatafile` removed to workaround init issues     | ✅     |
+|                     | `setStickyFeatures`                                 | ✅     |
+|                     | `getRevision`                                       | ✅     |
+|                     | `getFeature`                                        | ✅     |
+|                     | `getBucketKey`                                      | ✅     |
+|                     | `getBucketValue`                                    | ✅     |
+|                     | `isReady`                                           | ✅     |
 |                     | `refresh`                                           |        |
 |                     | `startRefreshing`                                   |        |
 |                     | `stopRefreshing`                                    |        |
-|                     | `evaluateFlag`                                      |        |
-|                     | `isEnabled`                                         |        |
```

Comment:
we need to keep both `evaluateFlag` and `isEnabled` methods.

Reply:
I had it before, and then I removed it. I'm an idiot XD